build.gradle

```diff
@@ -19,23 +19,23 @@
 buildscript {
     ext {
         minSdkVersion = 14
-        targetSdkVersion = 28
-        compileSdkVersion = 28
+        targetSdkVersion = 29
+        compileSdkVersion = 29
 
-        androidGradlePluginVersion = "3.2.1"
+        androidGradlePluginVersion = "3.5.0"
         androidMavenGradlePluginVersion = "1.5"
-        androidxAnnotationVersion = "1.0.0"
-        androidxAppCompatVersion = "1.0.2"
-        androidxCoreVersion = "1.0.1"
+        androidxAnnotationVersion = "1.0.2"
+        androidxAppCompatVersion = "1.1.0"
+        androidxCoreVersion = "1.1.0"
         androidxLegacySupportV4Version = "1.0.0"
```

Comment:
I think this one is only declared and included [here](https://github.com/google/flexbox-layout/blob/update_to_29/demo-playground/build.gradle#L43), but never used.

Reply:
Done